src/Vlingo.Xoom/codegen/content/ClassFormatter.cs

```diff
@@ -8,7 +8,7 @@
 using System;
 using System.Linq;
 
-namespace io.vlingo.xoom.codegen.content {
+namespace Vlingo.Xoom.Codegen.Content {
     public class ClassFormatter {
 
         public static string qualifiedNameOf(string packageName, string className) {
```

Comment:
Just highlighting it again but still could you keep names of methods, enums an properties like in idiomatic C#

Reply:
Fixed